### PR TITLE
Fixed broken directory output

### PR DESCRIPTION
**Pull request by clivewalkden**
## This PR is a:

- [ ] New topic
- [ ] Content update
- [x] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will fix the page output as it currently cuts off early due to unencoded reference <Magento_Blank_theme_dir>. 
Currently about half the page is missing due to this unencoded text.

## Additional information

List all affected URL's 

- https://devdocs.magento.com/guides/v2.1/frontend-dev-guide/templates/template-email.html
- https://devdocs.magento.com/guides/v2.2/frontend-dev-guide/templates/template-email.html

I haven't been able to get Jekyll to output the content for testing, will try on a different machine later.

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=2839) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=2839) before we can accept your contribution.<br/>

**Comment by magento-engcom-team** [comment]
@clivewalkden thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by dshevtsov** [comment]
The breaking symbols were introduced in #2789

**Comment by shrielenee** [comment]
Hey @clivewalkden - I pulled down your branch and viewed it locally and am not seeing a change. Test it out and let me know what you're seeing?

File: guides/v2.1/frontend-dev-guide/templates/template-email.md [hunk 1]
```diff
@@ -139,55 +139,55 @@ The styles for emails are split into several different files.
...
           <p>Contains majority of styles for emails, including resets, layout, typography, and so on. Review the comments at the top of this file to understand how the styles in this file are split between the <code>email.less</code> and <code>email-inline.less</code> files.</p>
       </td>
     </tr>
     <tr>
       <td>
-          <p><code><Magento_Blank_theme_dir>/web/css/source/_email-extend.less</code></p>
+          <p><code>&lt;Magento_Blank_theme_dir&gt;/web/css/source/_email-extend.less</code></p>
       </td>
       <td>
           <p>This file is intended to be copied into your custom themes and edited directly. You can add new email styles or override existing ones. This should prevent having to copy the <code>_email-base.less</code> file into your custom theme. See the <code><Magento_Luma_theme_dir>/web/css/source/_email-extend.less</code> file for example usage.</p>
```
Comment (bdenham) [inline guides/v2.1/frontend-dev-guide/templates/template-email.md]: Escape <Magento_Luma_theme_dir> as well.

File: guides/v2.1/frontend-dev-guide/templates/template-email.md [hunk 2]
```diff
@@ -139,55 +139,55 @@ The styles for emails are split into several different files.
...
       </td>
     </tr>
     <tr>
       <td>
-          <p><code><Magento_Blank_theme_dir>/web/css/source/_email-variables.less</code></p>
+          <p><code>&lt;Magento_Blank_theme_dir&gt;/web/css/source/_email-variables.less</code></p>
       </td>
       <td>
           <p>The <code>_email-base.less</code> file uses a number mixins from the Magento UI library. If you want to change any of the styles output by those mixins, you can set the value of any of the variables those mixins uses in this file. See the <code><Magento_Luma_theme_dir>/web/css/source/_email-variables.less</code> file for example usage.</p>
```
Comment (bdenham) [inline guides/v2.1/frontend-dev-guide/templates/template-email.md]: Escape <Magento_Luma_theme_dir>

**Comment by clivewalkden** [comment]
Latest commit is tested on a different machine which does compile Jekyll correctly. I've also updated some images:
- /common/images/email_create_template.png
- /common/images/email_choose-template.png
- /common/images/email_insert_variable.png

to add the 21 reference before the extension which seemed to be missing.

**Pull request merged**